### PR TITLE
Issue119 Concept-module rules are too strict

### PR DESCRIPTION
**Pull request by emmurphy1**
See [Issue 119 ](https://github.com/redhat-documentation/modular-docs/issues/119#)for the discussion.

### CODE REVIEW

**Review by rolfedh — APPROVED** [review]

**Comment by yzimmerm** [comment]
= Latency improvement by minimizing memory swapping

[role="_abstract"]
Swapping pages to disk can introduce latency in any environment. To ensure low latency, ensure that you have enough memory in your systems, so that swapping is not necessary. Always size the physical RAM as appropriate for your application and system.

Use the [command]`vmstat` command to monitor memory usage and watch the [parameter]`si` (swap in) and [parameter]`so` (swap out) fields. It is optimal that they remain zero as much as possible.

[literal,subs="+quotes,verbatim"]
----
$ *vmstat*
procs  -----------memory---------- ---swap--- -----io---- --system-- ------cpu-----
 r  b   swpd   free    buff  cache    si   so    bi    bo   in   cs  us sy id wa st
 9  0 4186588 517872     28 4366932    0    1     4   207    0    1   9  3 88  0  0
----

File: modular-docs-manual/content/topics/module_guidelines-concept.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ The concept body describes the subject of the concept module.
 Apart from paragraphs, you can use other AsciiDoc elements, such as lists, tables, or examples.
 Consider including graphics or diagrams to speed up the understanding of the concept.
 
-Do not include any instructions to perform an action, such as executing a command.
+Avoid including instructions to perform an action, such as executing a command.
 Action items belong in procedure modules.
 See also link:http://www.informationmapping.com/fspro2013-tutorial/infotypes/infotype2.html[The Six Information Types] at _informationmapping.com_ for ways to present different types of conceptual information: principle, concept, structure, process, fact.
```
Comment (sterobin) [inline modular-docs-manual/content/topics/module_guidelines-concept.adoc:34]: Wording addition per meeting. Just a start. Feel free to tweak as needed.

```
... procedure modules. However, in some cases a concept or reference can include suggested actions when those actions are simple, are highly dependent on the context of the module, and have no place in any procedure module. In such cases, ensure that the heading of the concept or reference remains a noun phrase and not a gerund.

For example, see the following section for link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.9/html-single/developing_decision_services_in_red_hat_process_automation_manager/index#bound_variables_in_patterns_and_constraints[Bound variables in patterns and constraints] and other sections that follow it in the example document. These concept and reference modules contain actions that are not suitable for standalone procedure modules but are relevant actions to understand in the context of the concept or reference being described.
```

File: modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc [hunk 2]
```diff
@@ -31,7 +31,7 @@ The contents of a concept module give the user descriptions and explanations nee
 
 * Look at nouns and noun phrases in related procedure modules and assemblies to find the concepts to explain to users.
 * Explain only things that are visible to users. Even if a concept is interesting, it probably does not require explanation if it is not visible to users.
-* Do not include any instructions to perform an action, such as executing a command. Action items belong in procedure modules.
+* Avoid including instructions to perform an action, such as executing a command. Action items belong in procedure modules.
```
Comment (sterobin) [inline modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc:34]: Not sure how much of the above actual doc instruction to duplicate here in the template. Maybe the same instruction from above but exclude the example? Trying to be consistent and clear while not burdening the template too much.

**Review by ncbaratta — APPROVED** [review]
lgtm

**Review by fbolton — APPROVED** [review]

**Review by rolfedh — APPROVED** [review]

File: modular-docs-manual/content/topics/module_guidelines-concept.adoc [hunk 3]
```diff
@@ -29,8 +29,9 @@ The concept body describes the subject of the concept module.
...
 Consider including graphics or diagrams to speed up the understanding of the concept.
 
-Do not include any instructions to perform an action, such as executing a command.
-Action items belong in procedure modules.
+Avoid including action items. Action items belong in procedure modules. However, in some cases a concept or reference module can include suggested actions when those actions are simple, are highly dependent on the context of the module, and have no place in any procedure module. In such cases, ensure that the heading of the concept or reference remains a noun phrase and not a gerund. For example, see link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.9/html-single/developing_decision_services_in_red_hat_process_automation_manager/index#bound_variables_in_patterns_and_constraints[
+"Bound variables in patterns and constraints"] and the sections that follow it. These concept and reference modules contain actions that are not suitable for standalone procedure modules but are relevant actions to understand in the context of the concept or reference being described.
```
Comment (rkratky) [inline modular-docs-manual/content/topics/module_guidelines-concept.adoc:33]:
```suggestion
```

**Review by yzimmerm — APPROVED** [review]

File: modular-docs-manual/content/topics/module_guidelines-concept.adoc (hunk 3 above)
Comment (emmurphy1) [reply modular-docs-manual/content/topics/module_guidelines-concept.adoc]: @rkratky can you explain why you don't want to include the examples?This is for the guide and not the template. I think the examples are very useful. The concept is somewhat abstract and I am afraid that writers will misinterpret this exception and use if for more explicit action.

**Review by yzimmerm — APPROVED** [review]

**Review by Preeticp — APPROVED** [review]

**Pull request merged**